modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -131,8 +131,6 @@ export default {
         negativeLabel: 'Resume Editing',
         affirmativeLabel: 'Discard Changes'
       },
-      confirmingDiscard: false,
```

Comment:
I think we can remove `confirmContent` as well, fwiw

Reply:
Thanks.